modules/getting-started-web-console-granting-permissions.adoc

```diff
@@ -1,30 +1,40 @@
 // Module included in the following assemblies:
 //
-// * getting-started/openshift-web-console.adoc
+// * tutorials/dev-app-web-console.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="getting-started-web-console-granting-permissions_{context}"]
 = Granting view permissions
 
-{product-title} automatically creates a few special service accounts in every project. The default service account takes responsibility for running the pods. {product-title} uses and injects this service account into every pod that launches.
+{product-title} automatically creates several service accounts in every project. The `default` service account takes responsibility for running the pods. {product-title} uses and injects this service account into every pod that launches.
 
-The following procedure creates a `RoleBinding` object for the default `ServiceAccount` object. The service account communicates with the {product-title} API to learn about pods, services, and resources within the project.
+By default, the `default` service account has limited permissions to interact with the OpenShift API.
+
+As a requirement of the application, you must assign the `view` role to the `default` service account to allow it to communicate with the OpenShift API to learn about pods, services, and resources within the project.
 
 .Prerequisites
 
-* You are logged in to the {product-title} web console.
-* You have a deployed image.
-* You are in the *Administrator* perspective.
+* You have `cluster-admin` or project-level `admin` privileges.
 
 .Procedure
 
-. Navigate to *User Management* and then click *RoleBindings*.
+. Navigate to *User Management* -> *RoleBindings*.
 . Click *Create binding*.
 . Select `Namespace role binding (RoleBinding)`.
```

Comment:
`Namespace role binding (RoleBinding)` option is only visible/accessible for cluster admin users 
this is the page project-level admin user will see after click `Create binding`
![Screenshot 2025-04-10 at 1 54 14 PM](https://github.com/user-attachments/assets/5dd52ca6-a6db-4c3f-be3b-3eeb07dabd40)

This is the page cluster admin user sees
![Screenshot 2025-04-10 at 1 52 17 PM](https://github.com/user-attachments/assets/ffa06016-5e0b-45b4-9109-779a022cad32)

Reply:
Thanks! I think I'll just remove that step though because it is already selected by default for cluster admins, so there really isn't any action that needs to be taken here.